webapp/src/hooks/useCopyToClipboard.tsx

```diff
@@ -0,0 +1,9 @@
+const useCopyToClipboard = (setHasCopied: () => void) => {
+  const copy = (text: string) => {
+    navigator.clipboard.writeText(text)
+    setHasCopied()
+  }
+  return copy
+}
```

Comment:
What if we build this as a un utility function instead of a hook? The main reason is that it doesn't need to store anything or use any other hook to operate.
```suggestion
const copyToClipboard = (onCopy: () => void) => {
    navigator.clipboard.writeText(text)
    onCopy()
}
```

Reply:
Yup!, your argument make sense a lot, there is no need for a hook for this. I'll change it and also do the other changes you suggested.

Reply:
But here in this copyToClipboard function, we also need to add the text as parameter to access the text inside the func
`const copyToClipboard = (text:string, onCopy:()=>void)`, if there is possibility please edit your suggestion or else I'll make a new pull request!